src/App.tsx

```diff
@@ -18,56 +55,32 @@ export const App: React.FC = () => {
               type="text"
```

Comment:
According to the check-list  - save selected suggestion text to the input on click and close the list;